### ISSUE TITLE
Error when redirecting

### ISSUE THREAD

**Issue by aerni** [jonassiewertsen/statamic-livewire#16]
Redirecting to another page within a Livewire component throws this error:

```
Jonassiewertsen\Livewire\Tags\Livewire::wildcard(): Return value must be of type string, null returned
```

You can reproduce this using this code:

```php
public function mount()
{
    return redirect()->to('/');
}
```

**Versions:**
Statamic 3.2.16 Pro
Laravel 8.64.0
PHP 8.0.12
jonassiewertsen/statamic-livewire 2.7.0

**Comment by Beaudinn** [comment]
#17

**Comment by jonassiewertsen** [comment]
Should be fixed in Version 2.7.1

 https://github.com/jonassiewertsen/statamic-livewire/releases/tag/v2.7.1

**Comment by jonassiewertsen** [comment]
Thanks @Beaudinn
